docs/mnist_example/mnist.cpp

```diff
@@ -272,7 +272,7 @@ static float img_data[] = {-0.4242129623889923f, -0.4242129623889923f,
 int main() {
   // Create an input tensor list of 1 tensor.
   int inputNum = 1;
-  OMTensor **inputTensors = (OMTensor **)malloc(inputNum * sizeof(OMTensor *));
+  OMTensor *inputTensors[inputNum];
```

Comment:
Since we are changing this file, might as well add the `omTensorListDestroy(tensorListOut)` at the end even though it's not strictly necessary in this case. But it can serve as a reminder to the users that they are responsible for freeing the memory for the output tensor list, deep or shallow.